app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -94,8 +95,12 @@
     private boolean hasZoomedChart = false;
 
     private NumberFormat sgvFormatter;
-    private boolean mmolxl;
-    private boolean mmolxlDecimals;
+    private static boolean mmolxl;
+    private static boolean mmolxlDecimals;
+
+    public static long timeLastGoodSGV = 0;
+    public static short pumpBattery = 0;
+    public static int countUnavailableSGV = 0;
```

Comment:
your looking at out of date commits that github is showing - if you look at "Files Changed" link it should show the final code